modules/toxiproxy/src/main/java/org/testcontainers/containers/ToxiproxyContainer.java

```diff
@@ -86,7 +86,7 @@ public ContainerProxy getProxy(String hostname, int port) {
                     throw new IllegalStateException("Maximum number of proxies exceeded");
                 }
 
-                final Proxy proxy = client.createProxy("name", "0.0.0.0:" + toxiPort, upstream);
+                final Proxy proxy = client.createProxy(hostname, "0.0.0.0:" + toxiPort, upstream);
```

Comment:
```suggestion
                final Proxy proxy = client.createProxy(upstream, "0.0.0.0:" + toxiPort, upstream);
```

Reply:
Hi @bsideup! Thanks for the suggestion.

Happy to accept it, but I'm not quite sure why we would want to have the port as part of the name. Is the hostname not enough?